app/views/calculators/leave_pot_untouched/_form.html.erb

```diff
@@ -27,6 +27,7 @@
   <script>
     window.dataLayer = window.dataLayer || [];
     dataLayer.push({
+      'event': 'CalculatorResult',
```

Comment:
Do you think we should call it `CalculatorResult`?

I'm inclined to rename the partials as we've adopted the path `/foo-calculator/estimate`.